test/test_05_01.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime, timedelta
+
+from hm_1.tasks.task_05_01 import Homework, Student, Teacher
+
+
+def test_class_homework_atributes_saved():
+    attributes = (
+        "Learn function",
+        timedelta(days=5),
+        datetime(2019, 1, 12, 15, 44, 35),
+    )
+    assert list(Homework(*attributes).__dict__.values()) == list(attributes)
+
+
+def test_class_homework_method_is_active_returns_true():
+    attributes = (
```

Comment:
Do you really need attributes?

Reply:
I thought it's more readable, but it's extra and I need to remove it?

Reply:
Than this?
```python
hw = Homework(
        "Learn function",
        timedelta(days=5),
        datetime(2021, 4, 12, 15, 44, 35),
     )

assert hw.is_active() is True
```